src/Components/NotificationIntegrationModal/Components/NotificationIntegrationModal.jsx

```
@@ -182,43 +182,64 @@ const NotificationIntegrationModal = ({
   
-  const handleSave = () => {
-    //notifications array for selected integrations
-    const notifications = [...(monitor?.notifications || [])];
-    
-    // Get all notification types IDs
-    const existingTypes = activeNotificationTypes.map(type => type.id);
+  // In NotificationIntegrationModal.jsx, update the handleSave function:
+
+const handleSave = () => {
+  // Get existing notifications
+  const notifications = [...(monitor?.notifications || [])];
```

ESLint

[error] 'monitor.notifications' is missing in props validation

(react/prop-types)